app/controllers/sessions_controller.rb

```diff
@@ -1,16 +1,6 @@
 class SessionsController < ApplicationController
+  include SessionsHelper
```

Comment:
Used sessions_helper as module to dry up code, however we wonder whether a module that is used only by one controller was appropriate.

Reply:
Good question. @iandouglas, thoughts?

Reply:
I usually don't pull code out into a module until it's used in more than one place, then I look for ways to DRY up that code. I'd also typically make helper methods that return some sort of status and use that status to let the controller figure out what to do, like having it return if it succeeded or not, and if not, what error, and where to redirect the user, and let the controller actually set the flash message and do the redirection. In this case you have to do some digging to figure out what's going on.